src/version_set.cc

```diff
@@ -254,6 +261,7 @@ Status VersionSet::Apply(VersionEdit* edit) {
 
 void VersionSet::AddColumnFamilies(
     const std::map<uint32_t, TitanCFOptions>& column_families) {
+  ReadLock l(&mutex_);
```

Comment:
should hold write lock (if we are to use rwlock). This function modifies `column_families_`.

Reply:
pardon my mistake. So vset is not read-intensive-like any more.